src/IndividualMovie.js

```diff
@@ -9,7 +9,7 @@ const IndividualMovie = (props) => {
     return(
         <div className='IndividualMovie'>
             {/* <p>{id}</p> */}
-            <p>{title}</p>
+            <p className='Title'>{title}</p>
```

Comment:
I noticed you were using capitalized classNames.  I would still stick with kebab-case for classes.